fedora_messaging/message.py

```diff
@@ -210,28 +205,30 @@ def _build_properties(self, headers):
         )
 
     @property
-    def headers(self):
-        return self.properties.headers
+    def _headers(self):
```

Comment:
Maybe add a docstring stating that we would consider making this property public if there is a use case for that. Otherwise I'm not sure people will ask.

Reply:
Done, good idea